app/src/main/java/org/jellyfin/mobile/settings/SettingsFragment.kt

```diff
@@ -63,10 +65,22 @@ class SettingsFragment : Fragment() {
             titleRes = R.string.pref_video_player_type_title
             initialSelection = VideoPlayerType.WEB_PLAYER
             defaultOnSelectionChange { selection ->
+                externalPlayerChoicePreference.enabled = selection == VideoPlayerType.EXTERNAL_PLAYER
                 swipeGesturesPreference.enabled = selection == VideoPlayerType.EXO_PLAYER
                 backgroundAudioPreference.enabled = selection == VideoPlayerType.EXO_PLAYER
             }
         }
+        val externalPlayerOptions = listOf(
+            SelectionItem(ExternalPlayerType.MPV_PLAYER, R.string.external_player_mpv, R.string.external_player_mpv_description),
+            SelectionItem(ExternalPlayerType.MX_PLAYER_FREE, R.string.external_player_mx_player_free, R.string.external_player_mx_player_free_description),
+            SelectionItem(ExternalPlayerType.MX_PLAYER_PRO, R.string.external_player_mx_player_pro, R.string.external_player_mx_player_pro_description),
+            SelectionItem(ExternalPlayerType.VLC_PLAYER, R.string.external_player_vlc_player, R.string.external_player_vlc_player_description),
+        ).filter { isPackageInstalled(it.key) }.plus(SelectionItem(ExternalPlayerType.SYSTEM_DEFAULT, R.string.external_player_system_default, R.string.external_player_system_default_description))
+        if (!isPackageInstalled(appPreferences.externalPlayerApp)) appPreferences.externalPlayerApp = ExternalPlayerType.SYSTEM_DEFAULT
+        externalPlayerChoicePreference = singleChoice(Constants.PREF_EXTERNAL_PLAYER_APP, externalPlayerOptions) {
+            titleRes = R.string.external_player_app
+            enabled = appPreferences.videoPlayerType == VideoPlayerType.EXTERNAL_PLAYER
+        }
```

Comment:
Could you put the external player settings below the ones for ExoPlayer please? That way it matches the order of the video player type preference.
Please also apply this to the rest of the code.